engine/src/test/java/org/camunda/bpm/engine/test/api/authorization/ManagementAuthorizationTest.java

```diff
@@ -252,14 +252,17 @@ public void testTelemetryEnabledWithoutAutorization() {
 
   public void testTelemetryEnabledAsCamundaAdmin() {
     // given
+    disableAuthorization();
+    managementService.enableTelemetry(true);
+    enableAuthorization();
     identityService.setAuthentication(userId, Collections.singletonList(Groups.CAMUNDA_ADMIN));
 
     // when
-    managementService.enableTelemetry(true);
+    managementService.enableTelemetry(false);
 
     // then
     String telemetryPropertyValue = TelemetryHelper.fetchConfigurationProperty(processEngineConfiguration).getValue();
```

Comment:
Not directly related to the topic of this ticket: Why is there no API method to check if telemetry is enabled?

Reply:
I created CAM-12078 for this.